packages/remark-captions/__tests__/__snapshots__/index.js.snap

```diff
@@ -21,9 +21,10 @@ print('bla')
 exports[`captions should break lists 1`] = `
 "<figure><blockquote>
 <ul>
-<li>Bar
-Source: fc2</li>
+<li>Bar</li>
 </ul>
+</blockquote><figcaption>fc2</figcaption></figure>
+<figure><blockquote>
```

Comment:
Nice side effect of the new "blank lines between quotes create separate quotes" behavior.

Reply:
I had a doubt on this one, but the title "captions should break lists" led me into thinking that the current behaviour was the expected one.